### PR TITLE
24 suggestion component widget

### LINKED ISSUE #24: Suggestion Component Widget
**Is your feature request related to a problem? Please describe.**
As a user, I should be able to see suggestions on what to do depending on the air quality.

**Describe the solution you'd like**
There should be a different suggestion for each air quality range that should be displayed in the center widget of the main page.

### PR DESCRIPTION
**Pull request by lindakw**
Added a widgets folder and two switch case files for the Aqi Level and Aqi Text info/advice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "@types/react": "^18.0.20",
     "expo": "^46.0.10",
     "expo-location": "^14.3.0",
-    "react": "^18.1.0",
-    "react-native": "^0.70.1",
+    "react": "18.0.0",
+    "react-native": "0.69.6",
```
Comment (kmausolf) [inline package.json:20]: Hmmm, was there something in 0.70.1 that was causing problems?

**Review by Kacique — APPROVED** [review]
Looks good

File: screens/aqi_screen/widgets/aqiLevel.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    case (aqiNum <= 150):
+      return <Text style={styles.aqiLevel}>Unhealthy for Sensitive Groups</Text>
+    case (aqiNum <= 200):
+      return <Text style={styles.aqiLevel}>Unhealthy</Text>
+    case (aqiNum <= 300):
+      return <Text style={styles.aqiLevel}>Very Unhealthy</Text>
+    case (aqiNum >= 301):
+      return <Text style={styles.aqiLevel}>Hazardous</Text>
+    default:
+      return <Text style={styles.aqiLevel}>None</Text>
```
Comment (kmausolf) [inline screens/aqi_screen/widgets/aqiLevel.tsx:19]: Whoa.  I had no idea you could match against expressions in JS.  This is wild!

**Review by kmausolf — APPROVED** [review]
Very nice work on the air quality level selectors!

File: package.json (hunk 1 above)
Comment (lindakw) [reply package.json:20]: Not that I can recall. I may not have updated to the latest version.

**Pull request merged**